CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Next Release
 1.0.1
 ------------
 
+* Fix issue #13: do not log message when logAttributes.msg is undefined.
```

Comment:
Sorry to bring this up quite late, this line needs to be moved to the "Next Release" section. Since `1.0.1` has already been released this would not be in the correct spot. 

If not updated, I'll likely pull down this remote and merge into a branch and commit it that way to master closing out this pull request.